internal/index/indexoperations/index.go

```diff
@@ -33,7 +41,9 @@ type Index struct {
 // the base path of the index.
 func ListIndexes(path string) ([]Index, error) {
 	dirs, err := ioutil.ReadDir(path)
-	if err != nil {
+	if os.IsNotExist(err) {
+		return nil, nil
```

Comment:
nit: now that we ensure that `index/` is present on startup, this error case no longer needs to be treated in a special way.